lib/seahorn/BvOpSem2.cc

```diff
@@ -1488,8 +1509,29 @@ class OpSemVisitor : public InstVisitor<OpSemVisitor>, OpSemBase {
     }
 
     if (F.getName().equals("shadow.mem.global.init")) {
+      Expr memOut = m_ctx.mkRegister(inst);
+      Expr memIn = m_ctx.getRegister(*CS.getArgument(1));
+      m_ctx.read(memIn);
       setValue(inst, lookup(*CS.getArgument(1)));
-      WARN << "Skipping initialization of a global: " << inst << "\n";
+
+      m_ctx.setMemReadRegister(memIn);
+      m_ctx.setMemWriteRegister(memOut);
+
+      LOG("opsem.mem.global.init", errs() << "mem.global.init: " << inst << "\n";
+          errs() << "arg1: " << *CS.getArgument(1) << "\n";
+          errs() << "memIn: " << *memIn << ", memOut: " << *memOut << "\n";
+          );
+
+      Value *gVal = (*CS.getArgument(2)).stripPointerCasts();
+      if (auto *gv = dyn_cast<llvm::GlobalVariable>(gVal)) {
+        auto gvVal = m_ctx.getGlobalVariableInitValue(*gv);
+        if (gvVal.first) {
+          m_ctx.MemCpy(lookup(*gv), gvVal.first, gvVal.second, 4);
```

Comment:
alignment probably depends on the machine. This is the alignment of symbolic pointer corresponding to `gv`. 
Depends on what `galloc()` does. Probably good idea to make it `4` or even `8` aligned on all architectures. Sync with what `galloc()` does right now and comment about the relationship

Reply:
It looks like currently, galloc defaults to the word size of the machine and only considers an alignment when it is greater than that size. Updated and comment added to MemCpy